src/material.h

```diff
@@ -33,6 +33,33 @@ Vec3 reflect(const Vec3& v, const Vec3& normal)
 	return v - 2 * dot(v, normal) * normal;
 }
 
+
+bool refract(const Vec3& v, const Vec3& n, float ni_over_nt, Vec3& refracted)
+{
+    Vec3 v_normalized = unit_vector(v);
+    float dt = dot(v_normalized, n);
+    float discriminant = 1.0f - ni_over_nt * ni_over_nt * (1.0f - dt * dt);
+    
+    if (discriminant > 0)
+    {
+        refracted = ni_over_nt * (v_normalized - n * dt) - n * sqrt(discriminant);
+        return true;
+    }
+    else
+    {
+        return false;
+    }
+}
+
+
+float schlick(float cosine, float ref_idx)
```

Comment:
reading the book, I start to think that, maybe, instead of calling the refractive index: `ref_idx`, may be more intuitive using something like: `ior`.
What do you think?

Reply:
Yep, it could be a better name.